packages/client/src/components/measurements/MeasurementCdfInput.re

```diff
@@ -466,6 +466,15 @@ module Main = {
   };
 };
 
+module MarkdownPreprocessing = {
+
+  let addNewLines = description: string => {
+    let duplicateNewLines = Js.String.replaceByRe([%re "/[\\n]/g"], "\n\n", description);
+    let pruneCuatrupleLines = Js.String.replaceByRe([%re "/[\\n\\n\\n\\n]/g"], "\n\n", duplicateNewLines);
```

Comment:
"quatruple" -> "quadruple". 
Also, This would make it impossible for anyone who wanted 4 newlines, I assume?

Reply:
This seems like a decently hacky situation to me. I realize it's a tricky thing though.

Reply:
You can't have 4 newlines right now. Try it.

Reply:
(Fixed the "quatruple" -> "quadruple" typo)

Reply:
Does the solution seem less hacky to you given that the 4 newlines is not a problem?

Some other things I was worried about were:
- Ordered lists breaking if you added an additional newline
- Tables

But I checked for that (https://www.foretold.io/c/e351f8f3-cf9c-4787-a787-68257220084d/m/012b73bf-82d2-4c70-9c85-644f2685fc68), and it is not a problem.